robottelo/api/utils.py

```diff
@@ -130,6 +130,31 @@ def publish_puppet_module(puppet_modules, repo_url, organization_id=None):
     return cv.read()
 
 
+def import_puppet_module(sat):
+    """Download install and import puppet class. This is workaround for 6.10, there is no longer
+    content view
+
+    :param sat: satellite object
+    :return: puppet environment name,
+        environment name is likely to be searched in next steps of the test
+    """
+    env_name = 'test_environment'
+    class_parameters_package_name = 'robottelo-api_test_classparameters-0.2.0.tar.gz'
```

Comment:
Would it possible to pass the module package name (or perhaps list of names) as a parameter?
In different tests we are using different modules (or combination). Searching for `'author': 'robottelo'` I've found we use these:
- generic_1
- generic_2
- api_test_classparameters
- cli_test_classparameters
- ui_test_classparameters